public/js/test/integration/todomvc.js

```diff
@@ -49,6 +49,23 @@ function resumeDebugger() {
   cy.get(".right-sidebar").find(".resume").click();
 }
 
+function addTodo(todo) {
+  if (false) {
+    debuggee(() => {
+      let input = driver.findElement(By.id("new-todo"));
+      input.sendKeys("yo yo yo", Key.ENTER);
+    });
+  } else {
+    cy.window().then(win => {
+      win.injectAgent();
+      win.apiClient.evaluate(`
+        Agent.type("#new-todo", "hi")
+        Agent.type("#new-todo", "{enter}")
```

Comment:
this is a fairly simple API. that supports type, click, dblclick. It's somewhat clunky, but I think we might be able to replace this with cypress' API which would be significantly cleaner